README.md

```diff
@@ -539,27 +583,27 @@ Follow the following steps to use tracing with Ballerina.
    reporter.max.buffer.spans=1000
 ```
 
-- Run Jaeger docker image using the following command
+- Run Jaeger Docker image using the following command
 ```bash
    $ docker run -d -p5775:5775/udp -p6831:6831/udp -p6832:6832/udp -p5778:5778 \
    -p16686:16686 p14268:14268 jaegertracing/all-in-one:latest
 ```
 
-- Navigate to `restful-service/guide` and run the restful-service using following command 
+- Navigate to `restful-service/guide` and run the restful-service using the following command
 ```
-   $ ballerina run restful_service/ --config restful_service/ballerina.conf
+   $ ballerina run restful_service --config restful_service/ballerina.conf
 ```
 
-- Observe the tracing using Jaeger UI using following URL
+- Observe the tracing using Jaeger UI using the following URL
 ```
    http://localhost:16686
 ```
 
 ### Metrics
-Metrics and alerts are built-in with ballerina. We will use Prometheus as the monitoring tool.
-Follow the below steps to set up Prometheus and view metrics for Ballerina restful service.
+Metrics and alerts are built-in with Ballerina. We will use Prometheus as the monitoring tool.
```

Comment:
Minor corrections:
"Metrics and alerts are built-in with Ballerina. Let's use Prometheus as the monitoring tool here.
Follow the steps below to set up Prometheus and view metrics for the `restful_service`."